README.md

```diff
@@ -5,11 +5,74 @@
 The GeoIP Update program performs automatic updates of GeoIP2 and GeoIP Legacy
 binary databases. CSV databases are _not_ supported.
 
-This is the new version of GeoIP Update. If for some reason you need the
-legacy C version, you can find it
-[here](https://github.com/maxmind/geoipupdate).
+## Installation
 
-## Installation from source or Git
+We provide releases for Linux, OSX, and Windows. Please see the
```

Comment:
I think it is "macOS" now. Should we make it clear somehow that "darwin" == macOS/OSX? Maybe "macOS (darwin)" or something would help.

Reply:
Good catch.